modules/nw-dual-stack-convert.adoc

```diff
@@ -6,13 +6,28 @@ As a cluster administrator, you can convert your single-stack cluster network to
 
 Converting a single-stack cluster network to a dual-stack cluster network consists of creating patches and applying them to the cluster's network and infrastructure.
 
-If you need to add IPv6 Virtual IPs (VIPs) for API and Ingress services to an existing dual-stack-configured cluster, you need to patch only the cluster's infrastructure and not the cluster's network.
-
 [NOTE]
 ====
 Each patch operation that changes `clusterNetwork`, `serviceNetwork`, `apiServerInternalIPs`, and `ingressIP` objects triggers a restart of the cluster. Changing the `MachineNetworks` object does not cause a reboot of the cluster.
 ====
 
+If you need to add IPv6 virtual IPs (VIPs) for API and Ingress services to an existing dual-stack-configured cluster, you need to patch only the cluster's infrastructure and not the cluster's network.
+
+[IMPORTANT]
+====
+If you already upgraded your cluster to {product-title} 4.16 or later and you need to convert the single-stack cluster network to a dual-stack cluster network, you must specify an IPv4 network for API and Ingress services for your cluster in the YAML configuration patch file. This configuration ensures that IPv4 traffic exists in the same network interface as the default gateway.
+
+.Example YAML configuration file with an added IPv4 address block for the `machineNetwork` network
+[source,yaml]
+----
+- op: add
+  path: /spec/platformSpec/baremetal/machineNetworks/- <1>
+  value: 192.168.1.0/24
```

Comment:
I think we want to add the IPv6 network here.  We should already have an IPv4 machineNetwork.

Reply:
Nope, that's not the case. If your cluster was born as pre-4.16, you never have Infrastructure CR with populated Machine Networks. That's why we need user to do it.

Only if you are born as 4.16 you have Infrastructure CR to contain correct Machine Networks from install-config.yaml

Reply:
Thanks, @mkowalski .

@rbbratta , would you be OK to approve this PR based on this discussion?